.github/workflows/ci_pullrequest.yml

```diff
@@ -236,3 +236,69 @@ jobs:
     needs:
       - "tests-postgres"
       - "tests-mysql"
+  # Build a quick container for validating the build succeeds (never break the builds)
+  # Push to ghcr.io/nautobot/nautobot-dev to be able to pull down if required for validation (see: #1683)
+  container_build_check:
+    name: "Check to see if we need to build a container"
+    needs: "integration-test"
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: "Check comments for magic statement"
+        id: comment_check
+        run: |
+          echo "::set-output name=magic_comment_present::"$(gh pr view https://github.com/nautobot/nautobot/pull/${{ github.event.number }} --json comments -q "[.comments[] | {body_trigger: .body | test(\"Please build a container.\"), author_assoc_trigger: .authorAssociation | test(\"MEMBER\")} | .body_trigger == true and .author_assoc_trigger == true] | any")
```

Comment:
IMO the magic comment should *look* like a magic comment like `[build containers]` or `!build_containers` or something like that. I'm considering aligning around how there are certain magic comments already accepted for things like GitHub commits (See: [`[skip ci]`](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs)).